index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" type="text/css" href="/src/lib/bootstrap/dist/css/bootstrap.css">
```

Comment:
If you are going to change your directory structure, make sure you include your bowerrc file or whoever pulls your code down will have to change your src paths.